### DISCUSSION TITLE
Configuration details (config.xml)

### DISCUSSION THREAD

**Discussion by gerardtost** [nuxsmin/sysPass#1815] [Show and tell]
Hi Rubén! sysPass is a great software, I like it very much!

I set some file extensions on config.xml file but the program requires mime types declarations.  So... Could you show us an example of mimes on config file?

XML tag:  `<filesAllowedMime></filesAllowedMime>`
Source:  https://github.com/nuxsmin/sysPass/blob/master/tests/res/config/config.xml

Thanks in advance.

**Comment by nuxsmin** [comment]
Hi Gerard!, I'm glad about you found sysPass useful :)

Regarding the mime types, they're loaded from [](https://github.com/nuxsmin/sysPass/blob/master/app/resources/mime.xml).

Regards
